WebDevStudios/wd_s
# Hotfix/hide svg sprite

### DESCRIPTION
Fixes an issue where the SVG sprite wasn't being hidden at the bottom of the page anymore.

This wraps the SVG in a div and hides the div, for now at least. I know we have some workflow changes happening in various branches so this is probably an okay stopgap for now, and maybe we can find a more solid way of doing this in one of those branches.

### SCREENSHOTS
![markup](https://d.pr/i/7AtvdW+)
![scss](https://d.pr/i/r8MTOC+)

### OTHER

### STEPS TO VERIFY
- Check out the branch
- Run `npm run build`
- Inspect the page and see that the SVG is hidden, so not creating any more page space

### DOCUMENTATION
> Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

I don't believe so.
